### ISSUE TITLE
Update pending property

### ISSUE THREAD

**Issue by michaelzangl** [foxbenjaminfox/vue-async-computed#28]
I currently use this plugin to get resources from the server. I want to display a loading indicator while content is refreshing / updating.

To do this, I'd like to have a property that indicates if a `asyncComputed` property is being recomputed. Something like:

```js
asyncComputed {
  someData: ...
}
...

this.someData // the computed value
this.someDataIsUpdating // implicit property, true while the computed value is updating
```

What do you think of this concept? Is it good to have those implicit properties or do you have a better idea? Should I implement it and create a PR?

**Comment by michaelzangl** [comment]
Created a PR for this: #29

**Comment by revelt** [comment]
This is good-one! vue-wait can do loaders but imho integrated solution, all-in-one, like this, is better

**Comment by michaelzangl** [comment]
Fixed with #45.
